standalone/runtime/src/lib.rs

```diff
@@ -1070,7 +1072,7 @@ impl pallet_registry::Config for Runtime {
 }
 impl pallet_mq::Config for Runtime {
 	type Event = Event;
-    type QueueNotifyConfig = ();
+    type QueueNotifyConfig = msg_routing::MessageRouteConfig<Self>;
```

Comment:
Indent?

Reply:
See the implementation of MessageRouteConfig.